resources/views/date/gig/listAttendances.blade.php

```diff
@@ -49,21 +49,32 @@
                                                     <td>{{ $user->abbreviated_name }}</td>
                                                     @foreach($gigs as $gig)
                                                         <?php $comment = $gig->hasCommented($user) ? $gig->getComment($user) : ''; ?>
-                                                        <?php $mark = $gig->hasCommented($user) ? '&nbsp;<i class="far fa-comment"></i>' : ''; ?>
+                                                        <?php 
+                                                            if($gig->hasCommented($user)){
+                                                                $gigid = $gig->id;
+                                                                $userid = $user->id;
+                                                                $mark = '&nbsp;<i class="far fa-comment comment-toggle" id="comment-toggle-'.$userid.'-'.$gigid.'" title="'.$comment.'"></i>';
+                                                                $mark .= '<div id="comment-'.$userid.'-'.$gigid.'"" style="display: none">'.$comment.'</div>';
+                                                                $mark .= '<script type="text/javascript">$(document).ready(function(){$("#comment-toggle-'.$userid.'-'.$gigid.'").click(function(){$("#comment-'.$userid.'-'.$gigid.'").toggle();});});</script>';
```

Comment:
As little JavaScript as possible in the "contents"-section and definitely no $(document).ready stuff. There is a "js"-section at the end of this file, where this can go and it already has a $(document).ready(...), where you can add your stuff to. Can use a class instead of an id and do something like:
`$('.hideable-comment').click(function() { $(this).toggle() })`

Reply:
fixed